cypress/e2e/webTables.cy.js

```diff
@@ -1,7 +1,54 @@
 /// <reference types='cypress' />
 
 describe('Web Tables page', () => {
-  it('', () => {
-
+  beforeEach(() => {
+    cy.visit('/webtables');
+  });
+  it('should display the total number of pages when new rows are added', () => {
+    cy.createWorkers(4, 11);
+    cy.get('.-totalPages').should('contain.text', '2');
+  });
+  it('should allow rows count selection', () => {
+    cy.createWorkers(4, 11);
+    cy.get('select').select('20 rows');
+    cy.get('.rt-td').should('contain.text', 'worker№' + 11);
+  });
+  it('should allow to add a new worker record', () => {
+    cy.createWorkers(4, 4);
+    cy.get('.rt-td').should('contain.text', 'worker№' + 4);
+  });
+  it('should allow to delete a worker record', () => {
+    cy.createWorkers(4, 4);
+    cy.get('#delete-record-4').click();
+    cy.contains('worker№' + 4).should('not.exist');
```

Comment:
not very good idea, because now it look like this
<img width="428" alt="image" src="https://github.com/mate-academy/qa_cypress_e2e_web_tables/assets/85580286/334f1b0f-c36e-4239-a88f-b848a8d4da39">

The best assertion here is to to get row that contains text workerN4 should not be visible